examples/Dockerfile/centos.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM dokken/centos-stream-9
+
+RUN yum -y install git g++ python3-pip curl zip unzip tar perl \
+    && pip install cmake
+
+WORKDIR /opt/tiledb
+COPY . .
+
+RUN cmake . -B build -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_MAKE_PROGRAM=make \
+      -DTILEDB_AZURE=ON -DTILEDB_S3=ON -DTILEDB_SERIALIZATION=ON -DTILEDB_TOOLS=ON \
+      -DCMAKE_INSTALL_PREFIX=/usr \
+    && cmake --build build \
+    && cmake --build build --target install-tiledb \
+    && ldconfig
```

Comment:
Maybe add an issue when it fails?  We [have an action for that](https://github.com/TileDB-Inc/github-actions/tree/main/open-issue) which i recently added to [another nightly](https://github.com/TileDB-Inc/TileDB-R/blob/master/.github/workflows/nightly.yaml) (and the main 'trick' is to pick a 'label' with an already existing label at the repo / else create the label as I did.